packages/gestalt/src/Badge.css

```diff
@@ -1,19 +1,27 @@
+:root {
+  --font-size-md: 12px;
+  --height: 24px;
+  --width: 42px;
+}
+
 .Badge {
   composes: antialiased sansSerif from "./Typography.css";
-  composes: fontWeightBold from "./Typography.css";
-  composes: blue from "./Colors.css";
   composes: inlineBlock from "./Layout.css";
-  border-radius: 4px;
-  border-style: solid;
-  border-width: 1px;
-  font-size: 10px;
-  line-height: 1;
+  composes: rounding2 from "./Borders.css";
+  composes: white from "./Colors.css";
+  composes: fontWeightNormal from './Typography.css';
+  line-height: 2;
```

Comment:
@denamwangi instead of adding `line-height`, can we use padding - especially `Thai` has issues with using line height? Been was playing around with it and this seems like a better approach:

```
display: inline-flex;
align-items: center;
white-space: nowrap;
font-size: 12px;
padding: 4px 8px;
min-height: 24px;
box-sizing: border-box;
```

Reply:
Ah yes- didn't realize this affected i18n. I'll take it out. For the changes you've added above with Annie's suggested new font size, padding and height- i'm going to hold off on that until there is consensus on the new standard and put that in a follow up diff but this approach looks great and i'll use for the next iteration.